lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3FileSystemAwsS3.java

```diff
@@ -49,6 +49,12 @@ protected void initEnvironment()
         bucket = environmentVariable("EMPTY_S3_BUCKET");
     }
 
+    @Override
```

Comment:
I'd argue that we should drop from `AbstractTestS3FileSystem` the method:

```
protected boolean isCreateExclusive()
    {
        return false;
    }
```

As a consequence, we should implement it to return `false` only on the s3 compliant implementation we're testing which is not supporting it.